### PR TITLE
Export env vars from scalability suite

### PR DESCRIPTION
**Pull request by wojtek-t**
Without exporting them, they didn't have any effect (were simply ignored in cluster/gce/config-test.sh)

### CODE REVIEW

**Comment by wojtek-t** [comment]
cc @gmarek

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -895,7 +895,10 @@ export KUBE_ENABLE_DEPLOYMENTS=${ENABLE_DEPLOYMENTS:-}
 export KUBE_ENABLE_EXPERIMENTAL_API=${ENABLE_EXPERIMENTAL_API:-}
 export MASTER_SIZE=${MASTER_SIZE:-}
 export MINION_SIZE=${MINION_SIZE:-}
+export MINION_DISK_SIZE=${MINION_SIZE:-}
```
Comment (gmarek) [inline hack/jenkins/e2e.sh]: s/MINION_SIZE/MINION_DISK_SIZE/

**Comment by gmarek** [comment]
One thing, LGTM otherwise.

File: hack/jenkins/e2e.sh (hunk 1 above)
Comment (wojtek-t) [inline hack/jenkins/e2e.sh]: Done.

**Comment by wojtek-t** [comment]
@gmarek - PTAL

**Pull request merged**